### PR TITLE
Topics are no longer hard coded in main.py

### PR DESCRIPTION
**Pull request by masonwoodford**
Implemented the topic sorting class into the main function.

### CODE REVIEW

**Comment by masonwoodford** [comment]
I also skip the first row now without hard coding tags. :)

File: browse/scripts/main.py [hunk 1]
```diff
@@ -36,12 +37,10 @@
...
-f.gen_out("../../database/db.csv", "../topics/places.md", "topic", "places")
-f.gen_out("../../database/db.csv", "../topics/science.md", "topic", "science")
-f.gen_out("../../database/db.csv", "../topics/daily.md", "topic", "daily")
-f.gen_out("../../database/db.csv", "../topics/home.md", "topic", "home")
-f.gen_out("../../database/db.csv", "../topics/furniture.md", "topic", "furniture")
-f.gen_out("../../database/db.csv", "../topics/technology.md", "topic", "technology")
+tp = TopicsParser()
+topics = tp.gen_topics("../../database/db.csv")
+for topic in topics:
+    f.gen_out("../../database/db.csv", '../topics/{}.md'.format(topic), "topic", "places")
```
Comment (sanketgarade) [inline browse/scripts/main.py:43]: shouldn't this be -

`f.gen_out("../../database/db.csv", '../topics/{}.md'.format(topic), "topic", '{}'.format(topic))`

Comment (sanketgarade) [reply browse/scripts/main.py:43]: otherwise the filter is hardcoded to "places" tag.

**Review by sanketgarade — CHANGES_REQUESTED** [review]

File: browse/scripts/main.py (hunk 1 above)
Comment (masonwoodford) [reply browse/scripts/main.py]: Yup you are right I did not notice that, should be fixed now.

Comment (sanketgarade) [reply browse/scripts/main.py]: Thanks 😊

**Pull request merged**